### ISSUE TITLE
List expense UG unclear and does not follow up in the examples

### ISSUE THREAD

**Issue by vimalapugazhan** [vimalapugazhan/pe#11]
![image.png](https://raw.githubusercontent.com/vimalapugazhan/pe/main/files/0f5de2dd-7104-4ebc-99e9-2c425288f6a0.png)

![image.png](https://raw.githubusercontent.com/vimalapugazhan/pe/main/files/a3bd42b7-a5f7-4603-a7c6-6439d4d9a20a.png)

The optional parameter is highlighted as [AMOUNT_TO_LIST] so it would make sense that "list /a" would be a valid input. However that is not the case as the example shows "list and "list /a 5" as valid. Instead including the"/a" in the optional field would make more sense.

list [/a AMOUNT_TO_LIST]

**Comment by nus-se-script** [comment]
# Team's Response
Resolving the inconsistent naming syntax in the UG will also resolve this naming issue

## The 'Original' Bug

[The team marked this bug as a duplicate of the following bug]

># Inconsistent naming convention used in the UG and the command line program
>
>**Note from the teaching team:** This bug was reported during the _Part II (Evaluating Documents)_ stage of the PE. **You may reject this bug if it is not related to the quality of documentation.**<hr>
>![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/bf59bb90-d6d7-4390-a7f7-550913a353ea.png)
>
>![image.png](https://raw.githubusercontent.com/nus-cs2113-AY2324S2/pe/master/files/40da0095-1b4f-4405-bc11-6837f07cc549.png)
>
>A few examples. Conventions also says that square brackets are optional and angled brackets are compulsory, so in that sense it makes it hard to follow
>
>
>
>
>
>
><hr>
><sub>[original: nus-cs2113-AY2324S2/pe-interim#2505] [original labels: severity.Medium type.DocumentationBug]</sub>
>
>
>

## Their Response to the 'Original' Bug

[This is the team's response to the above 'original' bug]

>Bug Chosen to address: the square brackets in the UG does not enclose the "/a" option in the list command
# Items for the Tester to Verify
## :question: Issue duplicate status

Team chose to mark this issue as a duplicate of another issue (as explained in the _**Team's response**_ above)

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
## :question: Issue severity

Team chose [`severity.Medium`]
Originally [`severity.Low`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
